website/content/en/docs/ansible/layout/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Ansible Based Operator Layout
+linkTitle: Layout
+weight: 300
+---
+
+{{% alert title="Warning" color="warning" %}}
```

Comment:
The doc `Information Flow` was in the main ansible menu and I do not think that it is some info that should be checked first by the users. It is moreover the architecture of the project, so I think that would be nice we have a new category  here to have it and the scaffold layout info doc.

Reply:
Information flow (IMO) is not really about the layout/scaffolding, but is an overview of the components of the ansible operator, which seem necessary for an operator developer to understand.

Reply:
I do not think that it more important the watches for example. So, I do not think that it should be after the quick-start.

Reply:
Solved